### PR TITLE
Dataload testing

### PR DESCRIPTION
**Pull request by franklu1**
Add Dataloads unit testing.

### CODE REVIEW

File: metron-streaming/Metron-DataLoads/src/test/java/org/apache/metron/dataloads/nonbulk/taxii/flatfile/SimplEnrichmentFlatFileLoaderTest.java [hunk 1]
```diff
@@ -0,0 +1,147 @@
+/**
```
Comment (cestella) [inline metron-streaming/Metron-DataLoads/src/test/java/org/apache/metron/dataloads/nonbulk/taxii/flatfile/SimplEnrichmentFlatFileLoaderTest.java]: Why is this file moved to org.apache.metron.dataloads.nonbulk.taxii from org.apache.metron.dataloads.nonbulk?

Comment (franklu1) [inline metron-streaming/Metron-DataLoads/src/test/java/org/apache/metron/dataloads/nonbulk/taxii/flatfile/SimplEnrichmentFlatFileLoaderTest.java]: You are right. It suppose to be in that folder. Mixed up while do the merge yesterday.


Frank Lu
ylu@hortonworks.com
(281) 468-7856

[page2image12144]

**Comment by franklu1** [comment]
close due to failed travis. Need relaunch travis for building.

File: metron-streaming/Metron-DataLoads/src/test/java/org/apache/metron/dataloads/nonbulk/flatfile/SimplEnrichmentFlatFileLoaderTest.java [hunk 2]
```diff
@@ -0,0 +1,147 @@
+/**
```
Comment (cestella) [inline metron-streaming/Metron-DataLoads/src/test/java/org/apache/metron/dataloads/nonbulk/flatfile/SimplEnrichmentFlatFileLoaderTest.java]: Filename is misspelled.  Please change to SimplEnrichmentFlatFileLoaderTest
